### ISSUE TITLE
Package scripts as in Linux distros

### ISSUE THREAD

**Issue by lucaderi** [ntop/ntopng#2953]
Currently scripts require multiple pieces. Example ntopng/scripts/callbacks/interface/flow/udp_unidirectional.lua has dependencies on
- i18n (that need to allow extensions to be included without having to modify the language script)
- scripts/callbacks/status_defs/status_udp_unidirectional.lua                                                                                                                                              
- scripts/callbacks/interface/flow/udp.lua                                                                                                                                                                 

In essence a user has to create multiple files and put them on various folders. This is not a nice practice as whenever a user wants to add/remove a script multiple files have to be copied. Instead it would be nice to package all these scripts on a single place where it is easy for a user to locate add/remove them. As initial solution a folder tree can work, but a more structured package would be nice to have.

This ticket depends on #2928

**Comment by emanuele-f** [comment]
Implemented in a3432e00e89f461492b033940bbb46f09bab2fb7
